modules/task/execution_test.go

```diff
@@ -114,21 +117,33 @@ func TestEnqueueWithoutRegister(t *testing.T) {
 }
 
 func TestConsumeWithoutRegister(t *testing.T) {
+	host, _ := service.NewScopedHost(service.NopHost(), "task", "hello")
+	_testScope = host.Metrics()
+	_globalBackend = NewInMemBackend(host)
+	_errorCh = _globalBackend.(*inMemBackend).ErrorCh()
```

Comment:
Cherami backend is doing testing through log messages and in-memory backend is doing testing through error channels. Should we unify it? In the end all you want is to know if a function was executed and what was the result, but methods are very different and we have an extra functionality(the error channels) just for testing.